.rubocop_todo.yml

```diff
@@ -7,12 +7,12 @@
 
 # Offense count: 8
 Metrics/AbcSize:
-  Max: 51
+  Max: 53
```

Comment:
Yes I think the same but I just want to see things *before* changing `.rubocop_todo.yml`. There could be something we can suggest to easily fix and if there is not then we suggest to just regenerate `.rubocop_todo.yml` :)